src/plugins/GUITestBase/src/tests/common_scenarios/Assembling/extract_consensus/GTTestsAssemblyExtractConsensus.cpp

```diff
@@ -0,0 +1,318 @@
+        : inputPaths(assembliesPath), algo(algo), keepGaps(keepGaps), outFile(outputFileName) {
+    }
+
+    void run(HI::GUITestOpStatus &os) override {
+        QWidget *const dialog = GTWidget::getActiveModalWidget(os);
+
+        // Dialog filling
+        std::transform(inputPaths.begin(), inputPaths.end(), inputPaths.begin(), [](const QString &path) {
+            return QFileInfo(path).absoluteFilePath();
+        });
+        GTLineEdit::setText(os, GTWidget::findExactWidget<QLineEdit *>(os, "Assembly widget", dialog),
+            inputPaths.join(';'));
+
+        GTComboBox::selectItemByIndex(os, GTWidget::findExactWidget<QComboBox *>(os, "Algorithm widget", dialog),
+            static_cast<int>(algo));
+
+        GTComboBox::selectItemByIndex(os, GTWidget::findExactWidget<QComboBox *>(os, "Keep gaps widget", dialog),
+            static_cast<int>(keepGaps));
+
+        GTLineEdit::setText(os, GTWidget::findExactWidget<QLineEdit *>(os, "Output file widget", dialog), outFile);
+
+        GTUtilsWizard::clickButton(os, GTUtilsWizard::Run);
+    }
+};
+
+GUI_TEST_CLASS_DEFINITION(sorted_bam_test_0001) {
+    const GTLogTracer lt;
+    const QString path = dataDir + "samples/Assembly/chrM.sorted.bam";
+
+    //  1. Select "Tools->NGS data analysis->Extract consensus from assemblies..."
+    //  2. Set "samples/Assembly/chrM.sorted.bam" as an input
+    //  3. Click "Run"
+    GTUtilsDialog::waitForDialog(os, new WizardFiller(
+        os, "Extract Consensus Wizard", new ExtractConsensusWizardScenario(QStringList() << path)
+    ));
+    GTMenu::clickMainMenuItem(os, QStringList() << "Tools"
+                                                << "NGS data analysis"
+                                                << "Extract consensus from assemblies...");
+
+    //  4. Wait for workflow finished
+    GTUtilsTaskTreeView::waitTaskFinished(os, 180000);
+    //  Expected state: There should be no errors in the log
+    CHECK_SET_ERR(!lt.hasErrors(), "Errors in log: " + lt.getJoinedErrorString());
+    //  There should be no notifications in the dashboard
+    CHECK_SET_ERR(!GTUtilsDashboard::hasNotifications(os), "Notifications in dashboard: " +
+        GTUtilsDashboard::getJoinedNotificationsString(os));
+
+    //  5. Return to workflow and call the Extract consensus wizard
+    GTUtilsWorkflowDesigner::returnToWorkflow(os);
+    //  6. Fill in the dialog :
+    //      {Algorithm} SAMtools
+    //      {Keep gaps} False
+    //      {Output file}
+    //     And click "Run"
+    GTUtilsDialog::waitForDialog(os, new WizardFiller(
+        os,
+        "Extract Consensus Wizard",
+        new ExtractConsensusWizardScenario(QStringList() << path, Algo::SAMTOOLS, false, "")
+    ));
+    GTToolbar::clickButtonByTooltipOnToolbar(os, MWTOOLBAR_ACTIVEMDI, "Show wizard");
+
+    //  7. Wait for workflow finished
+    GTUtilsTaskTreeView::waitTaskFinished(os, 180000);
```

Comment:
2 timeouts by 3 minutes in the same test.
Nightly tests have timeout equal to 4 minutes in total.
Consider moving this test to the Metagenomics suite or reduce timeouts or test data to make the test faster.